### PR TITLE
en: add description of task config `ignore-checking-items` (#293)

### PR DESCRIPTION
**Pull request by ti-srebot**
cherry-pick #293 to release-1.0

---

### What is changed, added or deleted? (Required)

add a description of task config `ignore-checking-items`, part of https://github.com/pingcap/docs-dm/pull/200

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v3.0 changes)
- [x] v2.0 (TiDB DM 2.0 versions)
- [x] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-dm/pull/177
- Other reference link(s): https://github.com/pingcap/docs-dm/pull/200

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts after cherry-picked

### CODE REVIEW

File: en/task-configuration-file-full.md [hunk 1]
```diff
@@ -30,8 +34,13 @@ meta-schema: "dm_meta"          # The downstream database that stores the `meta`
 remove-meta: false              # Whether to remove the `meta` information (`checkpoint` and `onlineddl`) corresponding to the task name before starting the replication task.
 enable-heartbeat: false         # Whether to enable the heartbeat feature.
 online-ddl-scheme: "gh-ost"     # Only "gh-ost" and "pt" are currently supported.
+<<<<<<< HEAD
```
Comment (TomShawn) [inline en/task-configuration-file-full.md:37]: @ireneontheway Please resolve the conflict according to https://docs.pingcap.com/zh/tidb-data-migration/stable/task-configuration-file-full#%E5%AE%8C%E6%95%B4%E9%85%8D%E7%BD%AE%E6%96%87%E4%BB%B6%E7%A4%BA%E4%BE%8B

Comment (ireneontheway) [reply en/task-configuration-file-full.md]: done

File: en/task-configuration-file-full.md [hunk 2]
```diff
@@ -31,7 +35,8 @@ remove-meta: false              # Whether to remove the `meta` information (`che
 enable-heartbeat: false         # Whether to enable the heartbeat feature.
 online-ddl-scheme: "gh-ost"     # Only "gh-ost" and "pt" are currently supported.
 case-sensitive: false           # Whether schema/table is case-sensitive.
-clean-dump-file: true           # Whether to clean up the files generated during data dump. Note that these include `metadata` files. New in v1.0.7.
+ignore-checking-items: []       # No element, which means not to disable any checking items.
+clean-dump-file: true           # Whether to clean up the files generated during data dump. Note that these include `metadata` files.
```
Comment (csuzhangxc) [inline en/task-configuration-file-full.md:39]: `New in v1.0.7.` lost?

**Review by csuzhangxc — COMMENTED** [review]
rest LGTM

File: en/task-configuration-file-full.md (hunk 2 above)
Comment (TomShawn) [reply en/task-configuration-file-full.md:39]: Maybe not. The Chinese version does not have this `New in v1.0.7.`, nor in the PR from which this PR is cherry-picked from (https://github.com/pingcap/docs-dm/pull/293).

Comment (TomShawn) [inline en/task-configuration-file-full.md:39]:
```suggestion
clean-dump-file: true           # Whether to clean up the files generated during data dump. Note that these include `metadata` files. New in v1.0.7.
```

**Review by TomShawn — APPROVED** [review]
LGTM

File: en/task-configuration-file-full.md (hunk 2 above)
Comment (TomShawn) [reply en/task-configuration-file-full.md]: Yes. Seems only v1.0 have this.

**Pull request merged**